std/StandardToken.sol

```diff
@@ -3,20 +3,20 @@ pragma solidity ^0.4.0;
 import "./Token.sol";
 
 contract StandardToken is Token {
-	uint256 public totalSupply;
```

Comment:
Then I'll revert the change in the `Token` contract, and I'll allow public state variables to shadow public interface functions.